mava/utils/lp_utils.py

```diff
@@ -86,6 +88,24 @@ def partial_kwargs(function: Callable[..., Any], **kwargs: Any) -> Callable[...,
     return functools.partial(function, **kwargs)
 
 
+def termination_fn(
+    parameter_server: SystemParameterServer,
+) -> None:
+    """Terminate the process
+
+    Args:
+        parameter_server: SystemParameterServer in order to get main pid
+    """
+    if parameter_server.store.manager_pid:
+        # parent_pid: the pid of the main thread process
+        parent_pid = parameter_server.store.manager_pid
```

Comment:
I can't seem to find where `parameter_server.store.manager_pid` is set in this PR? Was it added in another PR?

Reply:
In the `launcher.py` the parameter_server will get a copy from the builder store and within this store it will receive the manager_pid that was initialized in `on_building_init_end`

Reply:
Ah yes, ok makes sense, thanks.